File: MobileApp/app/src/main/java/com/example/professorperformaceevaluation/model/SchoolPeriod.java
package com.example.professorperformaceevaluation.model;

import java.util.Date;

public class SchoolPeriod {

    private int idSchoolPeriod;
    private Date startDate;
    private Date endDate;

    public SchoolPeriod() {
    }

    public int getIdSchoolPeriod() {
        return idSchoolPeriod;
    }

    public void setIdSchoolPeriod(int idSchoolPeriod) {
        this.idSchoolPeriod = idSchoolPeriod;
    }

    public Date getStartDate() {
        return startDate;
    }

    public void setStartDate(Date startDate) {
        this.startDate = startDate;
    }

    public Date getEndDate() {
        return endDate;
    }

    public void setEndDate(Date endDate) {
        this.endDate = endDate;
    }

}

File: DesktopApp/Model/SchoolPeriod.cs
using Newtonsoft.Json;
using System;

namespace ProfessorPerformanceEvaluation.Model
{
    public class SchoolPeriod
    {
        [JsonProperty("idSchoolPeriod")]
        public int IdSchoolPeriod { get; set; }
        [JsonProperty("startDate")]
        public DateTime StartDate { get; set; }
        [JsonProperty("endDate")]
        public DateTime EndDate { get; set; }
    }
}